sofiadinizms/vida-de-inseto
# Feature/level mix

### What I did

Finished the add of state and bindings in each level and add delay in level transitions 

### How to test

Git pull this branch
Run it on your iPhone
You need to be able to pass on each level, verify if after command completed it takes seconds until the next level shows up. 

If you managed to successfully complete all the task described above, then everything is working correctly
